streams/test-utils/src/test/java/org/apache/kafka/streams/TopologyTestDriverTest.java

```diff
@@ -385,10 +397,25 @@ public void shouldCloseProcessor() {
     @Test
     public void shouldThrowForUnknownTopic() {
         final String unknownTopic = "unknownTopic";
-        final ConsumerRecordFactory<byte[], byte[]> consumerRecordFactory = new ConsumerRecordFactory<>(
-            "unknownTopic",
-            new ByteArraySerializer(),
-            new ByteArraySerializer());
+        final byte[] nullValue = null;
+
+        testDriver = new TopologyTestDriver(new Topology(), config);
+        try {
+            pipeRecord(unknownTopic, new TestRecord<byte[], byte[]>(nullValue));
```

Comment:
use `final IllegalArgumentException exception = assertThrows(...)` instead of try-catch

Reply:
changed